src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -141,109 +140,88 @@ class WorkspaceContainer extends Component {
 }
 
 
-export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, showTabBar = true, queryparams }) => WrappedComponent => {
-  const WrappedClassComponent = toClass(WrappedComponent)
-
-  return ajaxCaller(class Wrapper extends Component {
-    constructor(props) {
-      super(props)
-      this.child = createRef()
-    }
-
-    static displayName = 'wrapWorkspace()'
-
-    render() {
-      const { workspaceError } = this.state
-
-      return workspaceError ? this.renderError() : this.renderSuccess()
-    }
+const WorkspaceAccessError = () => {
```

Comment:
Why have this as a function at all? Why not just start with a lowercase 'w' and just have it be assigned directly?

Reply:
Generally just leaning into Components as the unit of abstraction for chunks of content. Especially now that hooks are available, this is a good default form for things, since we could even add state if needed without refactoring the component or the call sites.

There's also the side benefit of not having to worry about load order dependencies. Here we call `Nav.getLink`, which I'm not 100% sure is safe to call at script load time.